### PR TITLE
feat(QAB-170): Migrate http micros-bdd steps

### PR DESCRIPTION
**Pull request by jordipuigbou**
Migrate HTTP high level private project steps according to [QAB-170](https://jira.elevenpaths.com/browse/QAB-170)

Add to Golium HTTP high level steps from private project because they are not project dependant and are useful to build more readable scenarios

**Documentation**

Not new documentation

**Test plan (required)**

- Added the corresponding unit tests for the new functions
- Added http/request and http/response acceptance test cases

### CODE REVIEW

File: steps/http/model/request.go [hunk 1]
```diff
@@ -40,7 +40,8 @@ func NewRequest(
 	request.Method = method
 	request.Endpoint = url + normalizeEndpoint(endpoint, endpoint, backslash)
 	request.RequestBody = nil
-	request.Path = path
+	request.Path = ""
```
Comment (ricardogarfe) [inline steps/http/model/request.go:43]: Path is not needed to be initialized in go

File: steps/http/model/request.go [hunk 2]
```diff
@@ -40,7 +40,8 @@ func NewRequest(
...
+	request.QueryParams = nil
```
Comment (ricardogarfe) [inline steps/http/model/request.go:44]: Also nil, QueryParams and RequestBody, unless we are giving nil a logic 👍🏽

File: test/acceptance/environments/local.yml [hunk 3]
```diff
@@ -1,7 +1,19 @@
 # HTTP settings
-url: https://httpbin.org
+http_bin_url: https://httpbin.org
```
Comment (ricardogarfe) [inline test/acceptance/environments/local.yml:2]: we can reorganize as config:
```
httpbin:
    url: https://httpbin.org
```
And use it as `[CONF:httpbin.url]`

Comment (jordipuigbou) [reply test/acceptance/environments/local.yml]: Done

File: steps/http/model/request.go (hunk 2 above)
Comment (jordipuigbou) [reply steps/http/model/request.go]: Done

**Review by ricardogarfe — APPROVED** [review]

**Review by sarmar11 — APPROVED** [review]

**Pull request merged**